tests/certification/pubsub/kafka/strimzi-kafka1-config/server.config

```diff
@@ -0,0 +1,104 @@
+##############################
```

Comment:
Could we only keep custom config for those?

Reply:
Most of this is needed unless I tried to build templating into flow and dynamically create each broker config from a single template. Is the added complexity worth it? These files should mostly be static.